az-digital/az_quickstart
# [WIP] Closes #1869: Alphabetical filter and search view for person

## Description
- Enables and configures az_alphabetical_listing module by default
- Makes az_person dependent on az_alphabetical_listing module
- Adds 4 new view displays to existing AZ person view
  - Grid with Search
  - Row with Search
  - Grid with Search and Alphabetical Filter
  - Row with Search and Alphabetical Filter

## Related issues
Closes #1869 

## How to test
1. Add person content
  a. Ensure Alphabetical Index Letter field is configurable
2. Select new views to add to pages
  a. _Note: Having search and search/filter views on one page will mess with filter styles a bit._ 


## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
